docs/en/3-api.md

```diff
@@ -456,6 +424,111 @@ $ cat stderr.txt
 BEMXJST ERROR: cannot render block b1, elem undefined, mods {}, elemMods {} [TypeError: Cannot read property 'undef' of undefined]
 ```
 
+### Using thirdparty libraries
+
+BEMTREE and BEMHTML allows you using thirdparty libraries as well as a global
+dependencies and different modular systems.
+
+For example:
+
+```js
+{
+    requires: {
+        'lib-name': {
+            globals: 'libName',           // Variable name from global scope
+            ym: 'lib-name',               // Module name from YModules
+            commonJS: 'path/to/lib-name'  // path to CommonJS library
+        }
+    }
+}
+```
+
+`lib-name` module will accessible in templates body like this:
+
+```js
+block('button').content()(function () {
+    var lib = this.require('lib-name');
+
+    return lib.hello();
+});
+```
+
+It’s necessary to specify all the modular systems for connecting the library.
+
+For example: you can specify dependencies only from global scope. In this case
```

Comment:
E.g. if you specify just `global scope` the library will only be available as global variable even though some module system will be present in runtime.